Python/ceval.c

```diff
@@ -912,7 +912,14 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
                 int frame_lasti = _PyInterpreterFrame_LASTI(frame);
                 PyObject *lasti = PyLong_FromLong(frame_lasti);
                 if (lasti == NULL) {
-                    goto exception_unwind;
+                    // gh-134163
+                    // If we can't allocate memory for lasti during exception handling,
```

Comment:
We typically follow a style like this:
```suggestion
                    // gh-134163: If we can't allocate memory for lasti during exception handling,
```

Reply:
tests case is very hard here, it must into repl and we can not use _testcapi for it because its itself

I will try to learn how to add it, but for me its kind of hard, can you give me some hint?

Reply:
at first I will try the eval loop..

Reply:
We're allowed to use `_testcapi` in pure-Python test cases, if that's what you're asking. It's late for me so I can look into finding a smaller repro without the REPL in the morning.

Reply:
Thank you very much and good night.
I will add news first